modules/dataframe/include/inviwo/dataframe/datastructures/column.h

```diff
@@ -52,15 +54,19 @@ class IVW_MODULE_DATAFRAME_API InvalidConversion : public Exception {
     virtual ~InvalidConversion() throw() {}
 };
 
+enum class ColumnType { Index, Ordinal, Categorical };
+
 /**
  * @brief pure interface for representing a data column, i.e. a Buffer with a name
  */
-class IVW_MODULE_DATAFRAME_API Column {
+class IVW_MODULE_DATAFRAME_API Column : public MetaDataOwner {
```

Comment:
Does not have too much to do with this PR, but we should use metadata as a member variable instead of using inheritance in my opinion